change/@uifabric-migration-2020-01-06-10-02-55-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
I think this could actually be "none" since this is a test-only change.

Reply:
Actually, maybe it's not test-only? I'm not sure based on the path, but it does seem to be in execution. I guess you'll let me know. :)